packages/drivers/routerlicious-driver/src/documentStorageService.ts

```diff
@@ -104,7 +91,8 @@ export class DocumentStorageService implements IDocumentStorageService {
     }
 
     public async readBlob(blobId: string): Promise<ArrayBufferLike> {
-        const iso = IsoBuffer.from(await this.read(blobId), "base64");
+        const blob = await this.readBlob(blobId);
+        const iso = IsoBuffer.from(blobToString(blob), "utf8");
 
         // In a Node environment, IsoBuffer may be a Node.js Buffer.  Node.js will
         // pool multiple small Buffer instances into a single ArrayBuffer, in which
```

Comment:
Any chance you can follow up here? I do not understand what this code is doing and why it's here. If such conversion is indeed required, why is it only here (in this driver) and not present in all other places? Should it be moved to more central place where we work with a bugger (like stringFromBuffer)? Tracking history and talking to whoever added it would be great! Alternative, feel free to open an issue (feel free to assign to me) as a follow up